app/src/main/java/com/github/se/project/ui/profile/CreateTutorSchedule.kt

```diff
@@ -58,83 +31,72 @@ fun AvailabilityScreen(
 
   Scaffold(
       topBar = {
-        TopAppBar(
-            title = { Text(text = "Availability") },
-            navigationIcon = {
-              IconButton(onClick = { /* Handle navigation */}) {
-                Icon(
-                    imageVector = Icons.AutoMirrored.Filled.ArrowBack,
-                    contentDescription = "Go back")
-              }
-            })
-      }) { paddingValues ->
+        Text(
+            text = "${profile.firstName}, show us your availabilities",
+            modifier = Modifier.padding(16.dp).testTag("welcomeText"),
+            style = MaterialTheme.typography.headlineSmall,
+            textAlign = TextAlign.Start)
+      },
+      content = { paddingValues ->
         Column(
             modifier =
                 Modifier.fillMaxSize()
+                    .padding(horizontal = 12.dp)
                     .padding(paddingValues)
-                    .padding(10.dp)
-                    .verticalScroll(rememberScrollState()),
-            horizontalAlignment = Alignment.CenterHorizontally) {
-              Text(
-                  text = "${profile.firstName}, show us your availabilities",
-                  fontSize = 24.sp,
-                  fontWeight = FontWeight.Bold,
-                  modifier = Modifier.padding(bottom = 8.dp))
-
+                    .testTag("availabilityScreen"),
+            verticalArrangement = Arrangement.spacedBy(16.dp)) {
               Text(
-                  text =
-                      "Finish your account creation by selecting the time slots you're available during the week:",
-                  fontSize = 16.sp,
-                  modifier = Modifier.padding(bottom = 8.dp))
+                  "Finish your account creation by selecting the time slots you're available during the week:",
+                  style = MaterialTheme.typography.bodyLarge,
+                  modifier = Modifier.testTag("instructionText"))
 
-              // Display the availability grid
               AvailabilityGrid(
```

Comment:
The 19h row looks off and is overlapping the 18h row when testing on my phone
![IMG_3014](https://github.com/user-attachments/assets/167880d9-1594-4633-a256-3901a5e3ae9a)